### ISSUE TITLE
⛔ 🐞 Bug Report: Tooltip (alt tag) appears on image with size = custom and caption = none

### ISSUE THREAD

**Issue by paulherzlich** [elementor/elementor#21249]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When you place an image element with Custom size and Caption set to None, you will see a tooltip with the image's Title from the media library. You should see nothing. There should be no tooltip. This problem does not happen when you choose another image size other than Custom. 

### Steps to reproduce

1) Place an image element. 2) Choose Custom size and Caption None. 3) View the page on the front end, and hover over the image. The image's Title in the media library appears as a tooltip. This should not happen. 

<img width="325" alt="Screenshot 2023-02-06 at 17 02 56" src="https://user-images.githubusercontent.com/7724692/217024574-7ee0ac13-0da6-4268-a063-785950732f9e.png">
<img width="925" alt="Screenshot 2023-02-06 at 17 03 23" src="https://user-images.githubusercontent.com/7724692/217024628-a2ad9564-3064-45ed-b650-2c03a8628815.png">

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.6.10
	PHP Version: 8.0.27
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://ele.maisonlamarque.com
	Home URL: https://ele.maisonlamarque.com
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 1024M
	Permalink Structure: /%postname%
	Language: en-GB
	Timezone: 0
	Admin Email: g.millar47470@hotmail.co.uk
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.2 Safari/605.1.15

== Active Plugins ==
	Elementor
		Version: 3.10.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.10.3
		Author: Elementor.com

	IONOS Performance
		Version: 2.0.1
		Author: IONOS


== Must-Use Plugins ==
	1&1 Product Subdomain
		Version: 1.1.0
		Author: 1&1


== Elements Usage ==
	
	footer : 1
		container : 4
		icon : 1
		progress-tracker : 1
		social-icons : 1
		text-editor : 3
	header : 3
		container : 11
		image : 1
		nav-menu : 3
		theme-site-logo : 2
	wp-page : 3
		button : 12
		container : 22
		heading : 33
		image : 3
		text-editor : 13


== Settings ==
	
	cpt_support: post, page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	css_print_method: internal
	unfiltered_files_upload: 1
	font_display: swap


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Active
	Accessibility improvements: Active by default
	Additional Custom Breakpoints: Active by default
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive
	Landing pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Active
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Inactive
	Loop: Inactive
	Form Submissions: Active by default
	Scroll Snap: Active by default


== Log ==
	
Log: showing 20 of 292023-01-09 20:04:50 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.2',
  'to' => '3.10.0',
)]
2023-01-09 20:04:51 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.9.2',
  'to' => '3.10.0',
)]
2023-01-19 15:23:02 [info] elementor::elementor_updater Started 
2023-01-19 15:23:02 [info] Elementor/Upgrades - _on_each_version Start  
2023-01-19 15:23:03 [info] Elementor/Upgrades - _on_each_version Finished 
2023-01-19 15:23:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.0',
  'to' => '3.10.1',
)]
2023-01-19 15:23:05 [info] elementor-pro::elementor_pro_updater Started 
2023-01-19 15:23:05 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-01-19 15:23:05 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-01-19 15:23:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.0',
  'to' => '3.10.2',
)]
2023-01-29 18:29:03 [info] elementor::elementor_updater Started 
2023-01-29 18:29:03 [info] Elementor/Upgrades - _on_each_version Start  
2023-01-29 18:29:04 [info] Elementor/Upgrades - _on_each_version Finished 
2023-01-29 18:29:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.1',
  'to' => '3.10.2',
)]
2023-01-29 18:29:04 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.1',
  'to' => '3.10.2',
)]
2023-01-29 18:30:04 [info] elementor-pro::elementor_pro_updater Started 
2023-01-29 18:30:04 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-01-29 18:30:04 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-01-29 18:30:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.2',
  'to' => '3.10.3',
)]
2023-01-29 18:30:04 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.2',
  'to' => '3.10.3',
)]

JS: showing 2 of 2JS: 2023-01-09 20:27:07 [error X 4][https://ele.maisonlamarque.com/wp-admin/load-scripts.php?c=1&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51823] undefined is not an object (evaluating 'a.replace') 
JS: 2023-01-26 20:27:34 [error X 1][https://ele.maisonlamarque.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.1:3:661303] this.model.isValidChild is not a function. (In 'this.model.isValidChild(C)', 'this.model.isValidChild' is undefined) 

PHP: showing 4 of 4PHP: 2023-01-25 14:17:16 [warning X 16][/homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-content/plugins/elementor/app/modules/import-export/module.php::628] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-25 14:17:18 [warning X 17][/homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-content/plugins/elementor/core/common/modules/ajax/module.php::172] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-25 17:33:44 [notice X 2][/homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-content/plugins/elementor/includes/template-library/sources/local.php::747] ob_end_clean(): Failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-01-26 19:42:13 [error X 1][/homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on null in /homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(5, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-content/plugins/elementor/core/common/modules/ajax/module.php(172): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /homepages/12/d917319034/htdocs/clickandbuilds/ELEMaisonLaMarque/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

				
````

</details>

**Comment by nicholaszein** [comment]
Hello @paulherzlich,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
- Alt tags are not the same as captions. They are an important accessibility component of web design, and should ALWAYS be present in any image tag. Their main purpose is to describe the image for visitors with visual impairments. They are also excellent for SEO, and many search engines use them for indexing pages correctly.
- Alt tags are currently not controlled by Elementor. They are part of WordPress and should be filled in when you upload your image, inside the image upload dialog box. If you don't fill in the alt tag, **<ins>YOUR BROWSER</ins>** will display the file name. So if you don't want the file name to appear, add alt tag information to your images in your Media Library.

### 👍 For now, that's all we can help with. 

If you think this is still an issue caused by Elementor, please feel free to open a new 🐞 [Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards
